src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -8449,6 +8449,12 @@
           "_meta": {
             "description": "Non-default value of setting."
           }
+        },
+        "enterpriseSearch:enableIndexPowerupsTab": {
```

Comment:
Note: this change came from adding the ui setting and running: `node scripts/telemetry_check --fix` per a failed CI job.